llvm/test/CodeGen/LoongArch/irtranslator-calllowering.ll

```diff
@@ -0,0 +1,16 @@
+; RUN: llc -O0 --mtriple=loongarch32 -global-isel -stop-after=irtranslator -verify-machineinstrs < %s \
```

Comment:
- Use standard options listed by `llc --help-hidden` and `FileCheck --help`. For example, `--global-isel` and `--check-prefix=`.
- Can we use utils/update_llc_test_checks.py to auto-gen the check lines?

Reply:
The standard option for GlobalISel is `-global-isel` with a single dash.

Reply:
I also think it should be the first option in the argument list